src/app/[locale]/(app)/(nav)/(time-selector)/stats/_components/ExtraStats.tsx

```diff
@@ -51,51 +37,51 @@ export default function Stats() {
     <div className="grid grid-cols-2 gap-2 px-2 py-4 desktop-container sm:grid-cols-3 sm:py-8 md:grid-cols-4">
       <StatCard
         value={formatNumber(notificationClicked(), { notation: "standard" })}
-        label="notifications clicked"
+        label="stats.notificationsClicked"
```

Comment:
For all of the labels, call `t` after importing it and adding the "use client" directive